test/parallel/test-vm-context.js

```diff
@@ -75,3 +75,14 @@ assert.throws(function() {
 // https://github.com/nodejs/node/issues/6158
 ctx = new Proxy({}, {});
 assert.strictEqual(typeof vm.runInNewContext('String', ctx), 'function');
+
+// https://github.com/nodejs/node/issues/10223
+ctx = vm.createContext();
+vm.runInContext('Object.defineProperty(this, "x", { value: 42 })', ctx);
```

Comment:
Blocks in JS Code look weird to me. I added some extra blank lines. OK like this?